extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -40,4 +43,129 @@ amp-story-subscriptions.i-amphtml-story-subscriptions-visible::before {
 amp-subscriptions-dialog.i-amphtml-story-subscriptions-dialog-wrapper {
   border-radius: 16px 16px 0px 0px !important;
   font-family: Poppins, sans-serif !important;
+  font-weight: 400 !important;
+  background: white !important;
+  color: black !important;
+}
+
+.i-amphtml-story-subscriptions-dialog-banner {
+  position: relative !important;
+  width: 100% !important;
+  border-radius: 16px 16px 0px 0px !important;
+  font-size: 24px !important;
+  text-align: center !important;
+  height: 44px !important;
+  background: #EDEDED !important;
+}
+
+.i-amphtml-story-subscriptions-dialog-content {
+  position: relative !important;
+  margin: 28px 28px 24px 28px !important;
+}
+
+.i-amphtml-story-subscriptions-publisher-text {
+  position: relative !important;
+  margin-bottom: 41px !important;
+}
+
+.i-amphtml-story-subscriptions-price, .i-amphtml-story-subscriptions-title,
+.i-amphtml-story-subscriptions-subtitle-first, .i-amphtml-story-subscriptions-subtitle-second {
+  overflow: hidden !important;
+  white-space: nowrap !important;
+  text-align: center !important;
+  display: flex !important;
+  justify-content: center !important;
+  flex-direction: column !important;
+}
+
+.i-amphtml-story-subscriptions-price {
+  width: 100% !important;
+  font-weight: 500 !important;
+  font-size: 24px !important;
+  margin-top: 28px !important;
+  letter-spacing: -0.4px !important;
+  height: 41px !important;
+}
+
+.i-amphtml-story-subscriptions-title {
+  width: 100% !important;
+  font-weight: 700, bold !important;
+  font-size: 20px !important;
+  letter-spacing: -0.25px !important;
+  height: 41px !important;
+}
+
+.i-amphtml-story-subscriptions-subtitle-first, .i-amphtml-story-subscriptions-subtitle-second {
+  width: 100% !important;
+  font-style: normal !important;
+  font-size: 16px !important;
+  letter-spacing: -0.25px !important;
+  height: 22px !important;
+}
+
+.i-amphtml-story-subscriptions-button-container {
+  display: flex !important;
+  justify-content: space-between !important;
+  flex-direction: column !important;
+  margin-bottom: 40px !important;
+  height: 92px !important;
+  font-family: Google Sans, sans-serif !important;
+}
+
+.i-amphtml-story-subscriptions-publisher-button-container, .i-amphtml-story-subscriptions-google-button-container {
+  display: flex !important;
+  justify-content: center !important;
+}
```

Comment:
If we remove these wrappers, `text-align: center` could be used on the parent most element and center the buttons and the text.